docs/content/Getting-Started/quickstart.md

```diff
@@ -11,34 +11,38 @@
 !!! tip "Estimated time to complete this example:" 
     ~4 minutes
    
-{%
-   include-markdown "../common-subs/required-packages.md"
-   start="<!--required-packages-start-->"
-   end="<!--required-packages-end-->"
-%}
 ## Setup Instructions
 
 Table of contents:
 
-1. [Install and run **KubeStellar**](#1-install-and-run-kubestellar)
-2. [Example deployment of Apache HTTP Server workload into two local kind clusters](#2-example-deployment-of-apache-http-server-workload-into-two-local-kind-clusters)
+0. [Install Prerequisites](#1-check-and-install-prerequisites)
```

Comment:
This PR already has a bunch of changes incrementing section numbers.  How about we make the text and the links consistent, both start at section number 1?

Reply:
I agree. I was *hoping* that I could start an ordered list with item 0, but discovered markdown always starts rendering with #1.   I fixed the ordinals in the links to be 1-5, but missed incrementing them in the md, because the rendering turned out 1-5 anyway.